src/sas/qtgui/Calculators/media/shape_2_sas_help.rst

```diff
@@ -0,0 +1,87 @@
+.. shape_2_sas_help.rst
+
+.. by J Krzywon, NIST, May 2025
+
+.. _Shape2SAS_Tool:
+
+Shape2SAS
+=========
+
+Description
+-----------
+
+This tool allows the user to build particles from a series of predefined shapes. By allowing particles to be placed in
+any location, or orientation relative to an origin, many unique structures can be constructed. Once a structure is
+created, the user can generate a theoretical data set that can be analysed, or an empirical fitting model can be
+generated to compare the structure to real data.
+
+Please note, the particles are constructed from a series of 'atoms' randomly distributed within the defined shape.
+Because of this randomness, creating the same particle two different times may result in subtle differences between
+scattering curves and slight differences in the resulting empirical models.
+
+The base shapes available to the user include spheres, cylinders, ellipsoids, elliptical cylinders, disks, cubes, cuboids,
+hollow spheres, hollow cubes, and cylindrical rings. Each of the particles can be individually modified in all dimensions,
+can be shifted relative to the origin, rotated relative to the particle centroid, and rotated relative to the coordinate
+volume origin. The scattering length density, relative to the medium the particle is in, can also be defined. If the
+particle is in air, this should be considered the scattering length density of the particle material.
+
+Using the Tool
+--------------
+
+.. image:: main_window.png
+
+When the tool is first selected from the menu option, the 'Build Model' tab will open. The user can select a shape and
+'Add' it to the table in the center of the window. The first row of the table is the particle shape, the second row is
+the scattering length density relative to the solvent, the next three rows are particle dimensions which vary between
+shapes, the sixth through eighth rows are to position the particle in space, the ninth through eleventh rows are for
+particle rotation about the particle centroid, and 12th through 15th rows are to rotate the particle relative to the
```

Comment:
As discussed above, I do not understand what these do. How can "particle rotation about the particle centroid" be in units of A? On the other hand, Degrees makes sense for rotation but I don't understand what is meant by " rotate the particle relative to the coordinate origin?"